jdaviz/vizcomponents/applications/cubeviz.py

```diff
@@ -35,13 +49,15 @@ def __init__(self, filename, vizapp):
         self._menu_bar_viewer.observe(self._on_change_menu_bar_viewer)
 
         # Add to menu bar
-        self._menu_bar = widgets.HBox([self._menu_bar_file, self._menu_bar_viewer])
+        self._menu_bar = v.Layout(children=[self._menu_bar_file, self._menu_bar_viewer])
```

Comment:
The issue is here - this should be a ``VBox`` - otherwise you are putting a vuetify layout inside a vbox inside a vuetify layout, which is causing the issues. So this should be:

```
        self._menu_bar = widgets.VBox([self._menu_bar_file, self._menu_bar_viewer])
```

Reply:
Thanks for the help @astrofrog, totally missed that I was doing that...  that's what I get for trying to change to many things at once, haha.